examples/playback_capture.rs

```diff
@@ -15,7 +15,7 @@ fn main() {
     let in_b = client.register_port("rust_in_r", j::AudioInSpec::default()).unwrap();
     let mut out_a = client.register_port("rust_out_l", j::AudioOutSpec::default()).unwrap();
     let mut out_b = client.register_port("rust_out_r", j::AudioOutSpec::default()).unwrap();
-    let process_callback = move |ps: &j::ProcessScope| -> jack::JackControl {
+    let process_callback = move |_: &j::WeakClient, ps: &j::ProcessScope| -> jack::JackControl {
```

Comment:
@rhetr

I created a new type for this shared client. It implements the `JackHandler` trait.

Does adding this new type make the API more complicated/confusing? If, so, I can use `ActiveClient`.